esx_service/volume_kv.py

```diff
@@ -58,6 +58,7 @@
 # The disk allocation format for vmdk
 DISK_ALLOCATION_FORMAT = 'diskformat'
 VALID_ALLOCATION_FORMATS = ["zeroedthick", "thin", "eagerzeroedthick"]
+CLONE_ADD_ALLOCATION_FORMAT = "delta" # Additional allocation format for clones
 DEFAULT_ALLOCATION_FORMAT = 'thin'
```

Comment:
Also agree with @brunotm here. 
In vmware speak the disks are either "delta disks" or "redo logs" . So I think "delta" in format is fine.  Also, "cloning", while usually applies to VM, is also already used as a "disk cloning" (https://kb.vmware.com/selfservice/microsites/search.do?language=en_US&cmd=displayKC&externalId=1028042) so we can keep using --option clone with "delta" disk IMO.